website/pages/en/index.js

```diff
@@ -58,35 +62,21 @@ const Logo = props => (
 
 const ProjectTitle = props => (
   <h2 className="projectTitle">
-    {siteConfig.title}
-    <small>{siteConfig.tagline}</small>
+    <img src="img/scalacenter2x.png" />
+    <em>Scala</em><i>fix</i>
+    <small>{siteConfig.tagline} for <em>Scala</em></small>
   </h2>
 );
 
-const PromoSection = props => (
-  <div className="section promoSection">
-    <div className="promoRow">
-      <div className="pluginRowBlock">{props.children}</div>
-    </div>
-  </div>
-);
 
 class HomeSplash extends React.Component {
+
   render() {
     let language = this.props.language || "";
     return (
       <SplashContainer>
-        {/* <Logo img_src={imgUrl('scalameta-logo.png')} /> */}
         <div className="inner">
           <ProjectTitle />
-          <PromoSection>
-            <Button href={docUrl("users/installation.html", language)}>
-              Documentation
```

Comment:
We should keep a "call to action" button. Currently, it's not clear that you should click on the front page.